Source/Scene/Primitive.js

```diff
@@ -108,7 +110,8 @@ define([
      * @param {Boolean} [options.cull=true] When <code>true</code>, the renderer frustum culls and horizon culls the primitive's commands based on their bounding volume.  Set this to <code>false</code> for a small performance gain if you are manually culling the primitive.
      * @param {Boolean} [options.asynchronous=true] Determines if the primitive will be created asynchronously or block until ready.
      * @param {Boolean} [options.debugShowBoundingVolume=false] For debugging only. Determines if this primitive's commands' bounding spheres are shown.
-     *
+     * @param {Boolean} [options.castShadows=true] Determines whether this primitive will cast shadows when shadow mapping is enabled.
```

Comment:
It's fine to just have this as a TODO in the roadmap for now if you want, but make sure to update CHANGES.md with all the public API changes before opening a pull request into master.